app/views/connections/_connection.html.haml

```diff
@@ -0,0 +1,11 @@
+- index = connection_counter + 1
+%tr(class="connection_#{index}")
+  %td
+    .pull-left
+      %img(src="#{gravatar_image_string(connection.guide)}")
+      = link_to "#{connection.guide_name}", user_path(connection.guide)
```

Comment:
This can be refactored to `= link_to connection.guide_name, connection.guide`.

`connection.guide_name` is a string anyway, no need to interpolate. `connection.guide` returns the user object and that will aim for users#show path - shorthand routes for the win :)